test/thermodynamics/internal_energy_change_via_amount_of_heat_and_work_done_test.py

```diff
@@ -0,0 +1,42 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (
+    assert_equal,
+    errors,
+    prefixes,
+    units,
+    Quantity,
+)
+from symplyphysics.laws.thermodynamics import (
+    internal_energy_change_via_amount_of_heat_and_work_done as first_law,
+)
+
+# Description
+## The amount of heat supplied to the system is 1 J and the work done by the system is 2 J.
```

Comment:
According to your convention, it is work done to the system.

Reply:
Still not fixed.
Here is quote from your description:
"The sign before work `W` is negative for work done _by_ the system"
So here it should be either `-2 J` or negative sign should be applied when passed to function.

Reply:
If the work is done by the system, the sign before the "delta W" should be negative, but the actual "delta W" is positive

Reply:
You do not need to play with description. You already have `minus` sign in formula so it should be _positive_ work done by the system. Or you can keep it negative but change the sign in formula to `plus`

Reply:
I propose making a separate law stating that `work_done_on_system = -1 * work_done_by_system` because this is a common source of confusion

Reply:
We have similar issues like force of action and reaction. I suggest to properly describe signs and that's all.